### ISSUE TITLE
App allowed invalid input on UNAVAIBLE DAY

### ISSUE THREAD

**Issue by jiaweiteo** [jiaweiteo/ped#6]
Command: `worker-add n/tommy3 hp/98103912 p/10 a/Singapore r/cleaner r/cashier u/Mon Tues FULL`

Added `u/Mon Tues FULL` in app and it recognised the input but only listed `Mon` as unavailble.
Depends on how you want to implement it. Can accept this input and show `Tues` as unavailable or can reject the input too.

![image.png](https://raw.githubusercontent.com/jiaweiteo/ped/main/files/6a447500-5c02-43a0-85ee-1b5368d67cb0.png)

**Comment by plosslaw** [comment]
You're right we should at least show some error message when invalid days are mixed in with valid days

Good catch!
